components/treelist/selection/cells.md

```diff
@@ -0,0 +1,253 @@
+---
+title: Cell Selection
+page_title: TreeList - Cells Selection
+description: Learn how to select cell in Blazor TreeList component. Explore the selected cells. Discover cell selection bevahior when combined with other TreeList features. Try the practical sample code for cell selection.
+slug: treelist-selection-cell
+tags: telerik,blazor,treelist,selection,cells
+position: 5
+---
+
+# Cell Selection
+
+The TreeList component supports [single or multiple cell selection]({%slug treelist-selection-overview%}#use-single-or-multiple-selection). You can select a cell with mouse click anywhere in the cell. You can access the collection of selected cells, use this collection and manipulate it. You can subscribe to selection events.
+
+## Basics
+
+To select a cell, click anywhere in it.
+
+To select a range of cells in one or more columns, hold the **Shift** key, while clicking on the first and last cell of the range. To select or deselect multiple cells that don't belong to a range, hold the **Ctrl** key.
```

Comment:
Generally, our style guide suggests monospace formatting for key names. [Source.](https://docs.telerik.com/style-guide/element-formatting)